editor/inspector_dock.cpp

```diff
@@ -553,7 +553,7 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	inspector->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	inspector->set_use_doc_hints(true);
 	inspector->set_hide_script(false);
-	inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));
+	inspector->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
 	inspector->set_use_folding(!bool(EDITOR_DEF("interface/editor/disable_inspector_folding", false)));
```

Comment:
As mentioned, I think this one should also get `s/editor/inspector/` for consistency.

Reply:
(and then drop "inspector" from the property name, so `interface/inspector/disable_folding`)

Reply:
Ok, I'll change.

Reply:
Changed.